scripts/tests/test_validate_docstrings.py

```diff
@@ -720,6 +726,13 @@ def test_bad_class(self):
         assert isinstance(errors, list)
         assert errors
 
+    @capture_stderr
+    def test_bad_class(self):
+        errors = validate_one(self._import_path(
+            klass='NoDocstrings'))['errors']
+        assert isinstance(errors, list)
```

Comment:
If we need this (though not sure we do) would be better to have a stronger assertion

Reply:
on the second thought this test is not strong enough, adding a test case in test_bad_generic function should be sufficient.

Reply:
an argument for leaving it in (with the fixed name) -- this class will give more informative error message in case NoDocstring class is removed from test source.